src/callbacks_step/analysis_dg2d.jl

```diff
@@ -231,6 +275,54 @@ function integrate(func::Func, u,
     end
 end
 
+function integrate_via_indices(func::Func, u,
+                               mesh::T8codeMesh, equations,
+                               solver::FV, cache, args...;
+                               normalize = true) where {Func}
```

Comment:
TODO: Relocate this routine and `calc_error_norms` to `analysis.jl` since it is dimension independent.